BEST_PRACTICES.md

```diff
@@ -0,0 +1,88 @@
+# Introduction
```

Comment:
I edit Markdown in Emacs, and it just kinda line wraps for me automatically. Sorry!